apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -197,8 +196,7 @@ function initializeBlocklyWrapper(blocklyInstance) {
     CdoRendererZelos,
     true /* opt_allowOverrides */
   );
-  registerAllContextMenuItems();
-  registerAllShortcutItems();
+  registerAllContextMenuShortcutItems();
```

Comment:
Could we initialize the plugin here and leave this line as `registerAllContextMenuItems()`?

Reply:
Yes - I agree this is a better place to initialize the plugin - thanks!